cmd/edge-proxy/main.go

```diff
@@ -140,7 +148,7 @@ func main() {
 
 	if enableHTTPSTunnel {
 		go func() {
-			err := server.StartHTTPSTunnel(httpsTunnelAddr, externalHTTPProxyURI, httpsTunnelTLSCert, httpsTunnelTLSKey, httpsTunnelUsername, httpsTunnelPassword)
+			err := server.StartHTTPSTunnel(httpsTunnelAddr, externalHTTPProxyURI, externalHTTPProxyCACert, httpsTunnelTLSCert, httpsTunnelTLSKey, httpsTunnelUsername, httpsTunnelPassword)
```

Comment:
Same as above. Maybe pass these parameters as an object, for example `HTTPSTunnelConfig`. This function has too many parameters.

Reply:
Done